### PR TITLE
[LLT-5476] Remove unnecessary Default impls

### PR DESCRIPTION
**Pull request by stalowyjez**
### Problem
There is a bunch implementations of Default trait which we may get rid of easily, so this PR tries to do that.

### Solution
Get rid of them

### :ballot_box_with_check: Definition of Done checklist
- [x] Commit history is clean ([requirements](../blob/main/docs/git_commit_messages_requirements.md))
- [x] README.md is updated
- [x] Functionality is covered by unit or integration tests

### CODE REVIEW

File: crates/telio-model/src/features.rs [hunk 1]
```diff
@@ -19,46 +20,26 @@ pub const DEFAULT_DIRECT_PERSISTENT_KEEPALIVE_PERIOD: u32 = 5;
...
-impl Default for FeaturePersistentKeepalive {
-    fn default() -> Self {
-        Self {
-            vpn: Some(DEFAULT_PERSISTENT_KEEPALIVE_PERIOD),
-            direct: DEFAULT_DIRECT_PERSISTENT_KEEPALIVE_PERIOD,
-            proxying: Some(DEFAULT_PERSISTENT_KEEPALIVE_PERIOD),
-            stun: Some(DEFAULT_PERSISTENT_KEEPALIVE_PERIOD),
-        }
-    }
-}
```
Comment (LukasPukenis) [inline crates/telio-model/src/features.rs:60]: Shouldn't these constants also be deleted in this PR ? :)

**Comment by mathiaspeters** [comment]
I think this is great, but I would like to have #730 merged before this, because I think the current set of tests is a bit lacking and I want to be sure this doesn't break anything

**Comment by stalowyjez** [comment]
@mathiaspeters sure :)

**Comment by LukasPukenis** [comment]
I left a comment about probably-unused constants, howevr I like the approach of inlining the defaults +1

**Comment by mathiaspeters** [comment]
+1 from me as well

**Review by packgron — APPROVED** [review]
+1 Awesome!

**Pull request merged**